doc/classes/Node2D.xml

```diff
@@ -54,15 +54,15 @@
 			<param index="0" name="delta" type="float" />
 			<param index="1" name="scaled" type="bool" default="false" />
 			<description>
-				Applies a local translation on the node's X axis based on the [method Node._process]'s [param delta]. If [param scaled] is [code]false[/code], normalizes the movement.
```

Comment:
I don't see the point in mentioning the 3D/2D equivalent of a method. I personally don't.